azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Improve Zangarmarsh gathering nodes and pooling

## Changes Proposed:
- Adds pools for all herbs and ores
- Removes the MASTER pools
- Create pool_pools for Khorium spawns at 2% and 3% chance

## Issues Addressed:

- Closes 

## SOURCE:

TBCMangos for new spawns, WotLK Classic for number of available spawns

## Tests Performed:

- 
- 


## How to Test the Changes:


1. Use Gathermate/Gatherer
2. Tap the nodes (don't pick them yet!)
3. Count all spawns, compare to the images below, see if they correlate
4. Pick them. See if it respawns later

## Known Issues and TODO List:


- [ ] Find the correct spawn timer. 90s is just too short for this type of pooling.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
